test/odoc_print/odoc_print.ml

```diff
@@ -204,9 +256,37 @@ let a_inp =
   let doc = "Input file." in
   Arg.(required & pos 0 (some file) None & info ~doc ~docv:"PATH" [])
 
+let a_short =
+  let doc = "Short output." in
+  Arg.(value & flag & info ~doc [ "short" ])
+
+let a_show_expansions =
+  let doc = "Show expansions in short output" in
+  Arg.(value & flag & info ~doc [ "show-expansions" ])
+
+let a_long_paths =
+  let doc = "Show long paths in short output" in
+  Arg.(value & flag & info ~doc [ "long-paths" ])
+
+let a_show_canonical =
+  let doc = "Show modules canonical reference in short output" in
+  Arg.(value & flag & info ~doc [ "show-canonical" ])
+
+let a_show_include_expansions =
+  let doc = "Show include expansions in short output" in
+  Arg.(value & flag & info ~doc [ "show-include-expansions" ])
+
+let a_show_removed =
+  let doc = "Show removed items in signature expansions in short output." in
+  Arg.(value & flag & info ~doc [ "show-removed" ])
```

Comment:
This one also seems unused. I think it should be true by default, the output of the tests wouldn't be changed currently.